### PR TITLE
Move Minkelite dependency to PM

### PR DESCRIPTION
**Pull request by kraman**
Connect to strongloop-internal/scrum-nodeops#563

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -4,33 +4,16 @@ var assert = require('assert');
...
 var debug = require('debug')('strong-mesh-models:server');
 var loopback = require('loopback');
-var MinkeLite = require('minkelite');
 
 /**
  * Factory method that returns a loopback server object. This object can be used
  * as express middleware.
  *
  * @param {ServiceManager} serviceManager
+ * @param {Minkelite} minkelite database for tracing
```
Comment (sam-github) [inline server/server.js]: nit: I'd call it a "Minkelite instance"... the DB is on disk.

**Comment by sam-github** [comment]
LGTM

**Comment by rmg** [comment]
This should definitely bump the major version.

**Comment by kraman** [comment]
@slnode test please

**Comment by kraman** [comment]
Bumped major version to 7.x

**Pull request merged**